packages/office-ui-fabric-react/src/utilities/keytips/KeytipManager.ts

```diff
@@ -40,22 +42,26 @@ export class KeytipManager {
    * @param persisted - T/F if this keytip should be persisted, default is false
    * @returns {string} Unique ID for this keytip
    */
-  public register(keytipProps: IKeytipProps, persisted: boolean = false): string {
+  public register(keytipProps: IKeytipProps, persisted: boolean = false, pollCallback?: () => IKeytipProps): string {
```

Comment:
>pollCallback?: () => IKeytipProps) [](start = 73, length = 34)

can you explain why the pollCallback is required, when it's returning (seemingly) the same this as what "keytipProps" is?

Reply:
Good point, we don't even need this as we already have a reference to the keytipProps